simulation_ws/src/aws_robomaker_simulation_common/src/aws_robomaker_simulation_common/route_manager.py

```diff
@@ -65,7 +65,9 @@ def __init__(self):
         self.route = []
 
         self.client = ActionClient(self, NavigateToPose, 'NavigateToPose')
+        time.sleep(5)
```

Comment:
Don't think this is necessary since `wait_for_server()` will already wait for the client to be ready.